### PR TITLE
Create instructor-debriefing-2015-09-08-15.html

### PR DESCRIPTION
**Pull request by raynamharris**
Draft blog post from Kate & Rayna's Sept 1 debriefing.  Needs to be revised and edited by Kate before publishing.

### CODE REVIEW

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<h2> Potential changes to the debriefing session format</h2>
+<p> Since March 2015, the debriefing sessions have served to purposes: 1) to debrief instructors of recent workshops, and 2) to give new (or even experienced) instructors an opportunity to hear who the lessons are going and get some tips. The impetus for including new instructors on the call came from <a href=" http://software-carpentry.org/blog/2015/03/instructor-debriefing-2015-03-10.html">a March debriefing session</a> when some novice instructors said they would have benefited from hearing instructor experiences BEFORE doing the instruction themselves. While the dual purpose debriefing session has be working, the mentoring committee is discussing desegregating the two. Stay tuned for more updates regarding specific mentoring opportunities for new instructors.</p>
```
Comment (fmichonneau) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: opportunity to hear *who* the lessons --> opportunity to hear **how** the lessons
dual purpose debriefing session has *be* working --> dual purpose debriefing session has **been** working
desegregating the two --> splitting them

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+<h2> Capstone sessions </h2>
+<p> A few of the recent workshops included a capstone or customized session at end. Thomas Guignard from the <a href=" https://mperignon.github.io/2015-08-06-cuboulder/">CU Bolder Workshop</a> shared his <a href=" https://mperignon.github.io/2015-08-06-cuboulder/">notes on OpenRefine</a> for librarians, which was based off of <a href="http://www.meanboyfriend.com/overdue_ideas/2014/11/working-with-data-using-openrefine/">this lesson from the British Library</a>. The <a href="http://andreww.github.io/2015-08-26-leeds/”>Leeds workshop </a> had a third day for students to find/analyze data related to environmental science (a method which <a href="https://software-carpentry.org/blog/2015/08/instructor-debriefing-2015-08-18-morning.html”>Kate previewed in a previous blog post</a>). The <a href="http://bsmith89.github.io/2015-08-27-unh”>UNH workshop</a> tried to incorporate a pandas-oriented capstone using data from the learners domain; this was an excellent idea, but the lesson had to be rushed to fit into a 50 minute time frame and was therefore not as successful as the instructors hoped. </p>
```
Comment (fmichonneau) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: at end --> at the end

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+<h2> Enhancing workshops with domain-specific examples</h2>
+<p> Sometimes we get lucky and the instructors are from the same domain as the learners. When this isn’t the case, we’ve seen that the helpers can be called on to motivate the material with domain specific examples. The <a href="http://dgasmith.github.io/2015-08-24-stanford/"> Stanford workshop</a>  learners were neuroscientists, so one of the helpers demoed a python notebook using data with which learners were familiar. The helpers also provided a <a href="http://wiki.scipy.org/NumPy_for_Matlab_Users”<“NumPy for Matlab Users” resource</a> to help make the Matlab to Python transition a little bit easier. Likewise, the librarians in the <a href=" https://mperignon.github.io/2015-08-06-cuboulder/">CU Bolder Workshop</a>  are most interested in cleaning data, data wrangling, and extracting information from datasets, so it can be a challenge to tailor SWC and Data Carpentry lessons to this audience. These instructors tailored the Pandas and SQL materials to accommodate the librarian learners. </p>
```
Comment (fmichonneau) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: the <a> tag for the NumPy for Matlab user link is incorrectly closed

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	
+<h2> No right or wrong way: lessons materials and Etherpad</h2>
+<p> Should I give students the full set of lesson materials before or after the workshop?  Should the learners or the helpers or the instructors run the Etherpad? These are hard questions to answer all options work and all have their own benefits. While some instructors worry that students will work ahead in the lessons and not pay attention, others have found that seeing the exercise instructions and full descriptions to be very helpful.  Likewise, some workshops have learners that work the Etherpad with collective enthusiasm while other learners prefer to only check the Etherpad when they need help. As co-instructors Rayna and Kate code along with the main instructor and paste commands in the Etherpad, which they have found to be a very efficient and useful for the students. Whichever route you chose, just be confident in your decision.</p>
```
Comment (fmichonneau) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: route you **choose**

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+<h2> Python reorganization </h2>
+<p> Byron Smith reorganized the flow of the python lesson but kept many of the same details from the lessons for the <a href="http://bsmith89.github.io/2015-08-27-unh”>UNH workshop</a>. Additionally, one learner suggested splitting up the monolithic Jupyter Notebook into smaller pieces, which would clean up the namespace and gave learners a more manageable set of "notes”. There have been many discussions on the python lessons lately, including these blogs.</p>
+
+<h2> Guacamole: A Dracula-Wolfman Alternative for Git</h2>
```
Comment (fmichonneau) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: that people can relate **to**

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
+¬¬¬¬¬ ---
```
Comment (rgaiacs) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: This should be only `---`.

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+<!-- start excerpt -->
+<p>
+The mentorship committee held their latest round of debriefing for instructors of recent workshops on Tuesday, September 1. Then, on Friday September 4th mentorship committee met for a regular bi-weekly meeting. Some comments from the committee meeting have made their way into this debriefing post.   
+</p>
+<!-- end excerpt -->
+
+<h2> New members on the mentoring committee</h2>
+<p> On August 22, Rainere Silva put out <a href=http://software-carpentry.org/blog/2015/08/join-mentoring-subcommittee.html>a call to join the mentorship committee</a> for members to help with the debriefing session, among other things. The new members include: Rayna Harris, Christina Koch, Sue McClatchy, Mariela Perignon, Phil Rosenfield, Michael Sarahan, and Belinda Weaver. </p>
```
Comment (rgaiacs) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: I was going to send a pull request about it today. =)

File: blog/2015/09/instructor-debriefing-2015-09-08-15.html (hunk 1 above)
Comment (rgaiacs) [inline blog/2015/09/instructor-debriefing-2015-09-08-15.html]: "Stay tuned for more updates regarding specific mentoring opportunities for new instructors." -> Could you also invite people to send us suggestions about that?

**Comment by rgaiacs** [comment]
@raynamharris Thanks for the pull request. Let me know when you address the comments.

**Comment by k8hertweck** [comment]
I made the changes recommended here and submitted a PR to Rayna's branch for her approval.

**Comment by raynamharris** [comment]
Thanks for everyone's comments, suggestions, and edits.  It ready to go now.

**Pull request merged**